frontend/src/pages/profile/components/user-activity/styles.module.scss

```diff
@@ -0,0 +1,73 @@
+.wrapper {
+  grid-area: activity;
+}
+
+.title {
+  margin: 30px 0 15px;
+}
+
+.activityWrapper {
+  display: flex;
+  max-width: 830px;
+  overflow-x: auto;
+}
+
+.days {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  padding-right: 6px;
+}
```

Comment:
Within that days are not aligned directly to `day` row - let' make it with grid area maybe
Same applicable for months showing

Reply:
Should I also add Sunday?
Now it looks like this:
![image](https://github.com/BinaryStudioAcademy/bsa-2023-writorium/assets/84803938/f59b58b9-25fd-45a3-a61e-858e1c3126b6)

Reply:
Yep, would look better I think